xbmc/video/guilib/VideoSelectActionProcessor.cpp

```diff
@@ -37,6 +39,18 @@ bool CVideoSelectActionProcessorBase::Process()
 
 bool CVideoSelectActionProcessorBase::Process(SelectAction selectAction)
 {
+  CVideoActionProcessorHelper procHelper{m_item, m_videoVersion};
+
+  if (!m_versionChecked && m_item->GetProperty("choose_video_version").asBoolean(true))
+  {
+    m_versionChecked = true;
```

Comment:
I suppose this is used to prevent endless recursion for the select dialog? As there is already the `choose_video_version` property, wouldn't it be sufficient to simply set this property to `false` once the dialog was shown? Just curious, no request to change anything.

Reply:
No. Again, separation of concerns! It's a property of the item. What we want to remember is a property of the processor. Unrelated things.